### ISSUE TITLE
Bar missing min, max and compute

### ISSUE THREAD

**Issue by mfirry** [andreaferretti/paths-scala-js#8]
The javascript version looks like 

``` js
export default function({data, accessor = id, width, height, min, max, gutter = 10, compute}) { /*... */}
```

Are we missing anything here?

**Comment by andreaferretti** [comment]
You are right for `min` and `max`. `compute` is only there for those ecosystems where the template rendering system does not allow computed properties (i.e., mustache) and is a pain to write a type for, so it will not be supported in the scala.js version
